src/components/Reference.js

```diff
@@ -118,7 +115,10 @@ export default class Reference extends React.Component {
               icon={<Icon className="material-icons">close</Icon>}
               label="CLOSE"
               value='x'
-              onClick={this.handleToggle} />
+              onClick={() => {
+                this.props.handleReferenceToggle();
+                this.setState({ value: 'a' });
```

Comment:
What is this line doing?

Reply:
(Line 120)

Reply:
The question is less so as to what it is doing but more so as why is it doing it.

Reply:
if this line wasn't included, closing the reference using the close button and then re-opening the reference would bring up [this page](https://imgur.com/34i0F7v), which shouldn't happen because there is no content on the non-existent "close" tab. idk why this happens but setting the reference page back to page "a" (i.e. the geometry page) when the close button is clicked fixes this.

Reply:
i could try to figure out why this only happens now, after i lifted the state of the reference component up to the header, but seeing as everything else works as expected besides this one weird bug, this was a super quick and easy fix that clears that up.

Reply:
Ah. I know why that's happening. It's because close says to open tab "x" which is non-existent.